docs/quick-start.md

```diff
@@ -272,7 +272,7 @@ scalaVersion := "{{site.SCALA_VERSION}}"
 libraryDependencies += "org.apache.spark" %% "spark-core" % "{{site.SPARK_VERSION}}"
 {% endhighlight %}
 
-For sbt to work correctly, we'll need to layout `SimpleApp.scala` and `simple.sbt`
+For SBT to work correctly, we'll need to layout `SimpleApp.scala` and `simple.sbt`
```

Comment:
sbt is lower-case. jar seems equally upper and lower case, so, fine to change but on the whole I tend to leave things like that. I think the other changes are OK but not sure I'd bother with things that aren't obviously wrong, like an Oxford comma.